packages/stateless/src/lib/handler/chat.handler.ts

```diff
@@ -13,7 +13,7 @@ export class ChatHandler {
       content = await this.message.template(data);
     }
 
-    if (!data.$wehookUrl) {
+    if (!data.$webhookUrl) {
```

Comment:
This is a great catch:
https://github.com/novuhq/novu/blob/2d3050a13b5dfc77b0d66e34ed2eac8a25299631/packages/stateless/src/lib/template/template.interface.ts#L33

I am wondering why it wasn't catched by the compiler. I am looking at you TypeScript. 👀